MacOSExample/package.json

```diff
@@ -3,26 +3,31 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "android": "react-native run-android",
-    "ios": "react-native run-ios",
-    "macos": "react-native run-macos",
+    "macos": "react-native-macos run-macos",
     "lint": "eslint .",
-    "start": "react-native start",
+    "start": "react-native-macos start",
     "test": "jest"
   },
   "dependencies": {
+    "@react-native-async-storage/async-storage": "^1.24.0",
+    "@react-native-community/slider": "^4.5.3",
     "@react-navigation/native": "^6.1.6",
     "@react-navigation/native-stack": "^6.9.12",
     "@react-navigation/stack": "^6.3.17",
+    "@swmansion/icons": "^0.0.1",
+    "babel-plugin-module-resolver": "^5.0.2",
     "hoist-non-react-statics": "^3.3.2",
     "invariant": "^2.2.4",
+    "patch-package": "^8.0.0",
+    "postinstall-postinstall": "^2.1.0",
     "react": "18.2.0",
-    "react-native": "0.72.4",
+    "react-native": "^0.73.9",
```

Comment:
Shouldn't `react-native-macos` 0.74 work with `react-native` 0.74?

Reply:
Fixed in [46a46ac](https://github.com/software-mansion/react-native-gesture-handler/pull/3055/commits/46a46ac95ce6fa6d33334c362cea374a0e972d6a)